schemas/input/csl-data.json

```
@@ -4,425 +4,477 @@
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "collection-editor": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "compiler": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "composer": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "container-author": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "contributor": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "curator": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "director": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "editor": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "editorial-director": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "executive-producer": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "interviewer": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "illustrator": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "organizer": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "original-author": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "performer": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "producer": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "recipient": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "reviewed-author": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
-        }
-      },
-      "translator": {
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/name-variable"
+      "required": ["type", "id"],
+      "additionalProperties": false
+    },
+    "related-bibitem": {
+      "title": "Related bib item",
+      "description": "Used to define related bibliographic data; value can either be a refid string, or an embedded item object",
```

```suggestion
      "description": "Used to define related bibliographic data; value can either be an item id or citekey string, or an embedded item object",
```

Do we ever use the term `refid` elsewhere?